.github/workflows/release_branch_issue.yml

```diff
@@ -32,7 +32,29 @@ jobs:
           echo 'TABLE<<EOF' >> $GITHUB_ENV
           echo "${table}" >> $GITHUB_ENV
           echo 'EOF' >> $GITHUB_ENV
-      - uses: JasonEtco/create-an-issue@v2
+      - name: Format Assignees and Update Markdown
+        run: |
+          # Fetch GitHub Group Members
```

Comment:
Are we updating the ticket template itself and checking in that change instead of assigning it to the correct users after creation?

Reply:
Good question! Yes, based on the workflow steps, it appears that the ticket template file itself (the markdown file) is being updated with assignees and committed back to the repository. However, if the goal is to only assign the correct users to the created ticket without modifying and committing the template file. Please let me know

Reply:
Also, if that's the case! should we remove the assignee line on the release_ticket md file?

Reply:
I think we should keep someone assigned to the ticket by default just in case something fails, but other than that I'm OK managing this by just assigning the ticket in CI to the group users

Reply:
Ok. I will remove 'the update md file' line as well as the 'Commit and Push Assignees Update to Markdown' step. This way it will just auto assign user from the GitHub group to the release ticket. We will keep the assignee on the md file as precaution. Is this ok?